test/lib/ansible_test/_data/completion/docker.txt

```diff
@@ -1,5 +1,5 @@
-default name=quay.io/ansible/default-test-container:2.9.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=collection
-default name=quay.io/ansible/ansible-base-test-container:1.7.0 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9 seccomp=unconfined context=ansible-core
+default name=quay.io/ansible/default-test-container:2.9.1 python=3.6,2.6,2.7,3.5,3.7,3.8,3.9,3.10 seccomp=unconfined context=collection
```

Comment:
You've changed the image version to one that does not exist. The CI will keep being broken for as long as it's not in the registry (quay) because ansible-test won't be able to download an unexisting container version. Its source lives in https://github.com/ansible/default-test-container and you may need to change it to include Python 3.10 alpha.

There are also other containers for integration tests, like Fedora, that live in https://github.com/ansible/distro-test-containers.

Reply:
This is correct: we do not currently have Python 3.10 in our default test container.